typespec-extension/src/code-model-builder.ts

```diff
@@ -305,6 +305,8 @@ export class CodeModelBuilder {
             extensions: {
               "x-ms-skip-url-encoding": schema instanceof UriSchema,
             },
+            // // make the logic same as TCGC, which takes the service default of host as client default
+            // clientDefaultValue: getDefaultValue(it.defaultValue),
```

Comment:
currently disable this line, not sure the status of corresponding TCGC behavior https://github.com/Azure/typespec-azure/issues/1030